mp/game/hl2sb/gameinfo.txt

```diff
@@ -1,8 +1,8 @@
 "GameInfo"
 {
-	game	"My First HL2MP Mod"
-	title	"HALF+LIFE'"
-	title2	"deathmatch mod"
+	game	"Half-Life 2 Sandbox"
+	title	" "
+	//title2	""
```

Comment:
OK, this is ready to be promoted to `master`, but could you `git rebase` and drop the changes from `gameinfo.txt`, please?

Thank you!

Reply:
https://github.com/Planimeter/hl2sb-2013/pull/2/commits/a4748dac1121ec1bfe93fe6e570172971edf364f